aspnetcore/fundamentals/metapackage-app.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Microsoft.AspNetCore.App metapackage for ASP.NET Core 2.1 and later
+author: Rick-Anderson
+description: The Microsoft.AspNetCore.App metapackage includes all supported ASP.NET Core and Entity Framework Core packages.
+manager: wpickett
+monikerRange: '>= aspnetcore-2.1'
+ms.author: riande
+ms.date: 09/20/2017
+ms.prod: asp.net-core
+ms.technology: aspnet
+ms.topic: article
+uid: fundamentals/metapackage-app
+---
+
+# Microsoft.AspNetCore.App metapackage for ASP.NET Core 2.1
+
+This feature requires ASP.NET Core 2.1 and later targeting .NET Core 2.1 and later.
+
+The [Microsoft.AspNetCore.App](https://www.nuget.org/packages/Microsoft.AspNetCore.App) metapackage for ASP.NET Core includes:
+
+* All supported packages by the ASP.NET Core team except those that contain third party dependencies.
+* All supported packages by the Entity Framework Core. 
+* Internal dependencies used by ASP.NET Core and Entity Framework Core.
+* 3rd-party dependencies used by ASP.NET Core and Entity Framework Core deemed necessary to ensure the major frameworks features function.
+
+All the features of ASP.NET Core 2.1 and later and Entity Framework Core 2.1 and later are included in the `Microsoft.AspNetCore.App` package. The default project templates targeting ASP.NET Core 2.1 and later use this package. We recommend applications targeting ASP.NET Core 2.1+ and Entity Framework Core 2.1+ use the `Microsoft.AspNetCore.App` package.
+
+The version number of the `Microsoft.AspNetCore.App` metapackage represents the ASP.NET Core version and Entity Framework Core version. The ASP.NET Core version is currently aligned with the .NET Core version, and we hope to continue version alignment in the future.
```

Comment:
The reason I mentioned the independence between the aspnetcore versions and the .net versions is because in 2.0 aspnet metapackage is no longer aligned with .net. i.e. the latest patch uses aspnetcore 2.0.8 and .net core 2.0.7. You would want to update the same section in the .All doc as well.